LondonBritishLibrary/orient/BLorient658.xml

```diff
@@ -57,60 +91,86 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-                                
-                                
+
                             </supportDesc>
                             <layoutDesc>
                                 <layout columns="3" writtenLines="29 31" cert="high">
                                 </layout>
                             </layoutDesc>
-                            
-                            
+
+
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is written in a fine hand, between <date notBefore="1670" notAfter="1682"/>, 
-                                    
+                                <desc>The manuscript is written in a fine hand, between <date notBefore="1670" notAfter="1682"/>,
+
                                 </desc>
-                                
-                                <date notBefore="1670" notAfter="1682"/>
-                                
+
+                                <date evidence="reign" notBefore="1670" notAfter="1682"/>
+
                             </handNote>
-                            
-                        </handDesc>   
-                        <additions>
-                            
-                            
+
+                        </handDesc>
+  <additions>
+<list>
+<item xml:id="a1">
+<locus target="#273r"/>
+<desc>A note stating that this volume was written by the order of king <persName ref="PRS10375Yohanne">John I</persName> for himself and his children when Sinodā was ʾabuna.
+To the names of John and <persName ref="PRS8278SablaWa">Sabla Wangel</persName> subsequently has been subsequently added that of their daughter <persName ref="PRS13245WalattaRufael">Walatta
+Rufāʾel</persName>.</desc>
+<q xml:lang="gez"><seg part="I">ነዋ፡ አዘዝነ፡ ንሕነ፡ አእላፍ፡ ሰገድ፡ ወበጸጋ፡ እግዚአብሔር፡ ዘተሰመይነ፡ ዮሐንስሃ፡ ያውግዙ፡ እንበይነ፡ ዛቲ፡ መጽሐፍነ፡ ዘአጽሐፍናሃ፡ ንሕነ፡
+በዘመነ፡ መንግሥትነ፡ ለበቍዔተ፡ ነፍስነ። ከመ፡ ኢያውፅኣ፡ እምቤተ፡ ውሉድነ፡ ዘወለድናሆሙ፡ በሥጋ። መኑሂ፡ እምሰብእ፡ እመሂ፡ ንጉሥ፡ ወእመሂ፡ መኰንን።</seg></q>
+</item>
+
+<item xml:id="a2">
+<desc>At the beginning of each month, at the foot of the page, is a name, probably that of a scribe.</desc>
+<q xml:lang="gez">ዘ<persName role="scribe" ref="PRS13257TawaldaMadhen">ተወልደ፡ መድኅን፡</persName> ዘ<persName role="scribe" ref="PRS13258ZamanfasQeddus">ዘመንፈስ፡ ቅዱስ፡</persName>,
+ዘ<persName role="scribe" ref="PRS13259TaklaMaryam">ተክለ፡ ማርያም፡</persName>, ዘ<persName role="scribe" ref="PRS13260Asaf">አሳፍ፡</persName>,
+ዘ<persName role="scribe" ref="PRS13261Giyorgis">ጊዮርጊስ፡</persName>, ዘ<persName role="scribe" ref="PRS13262DamaKrestos">ደመ፡ ክርስቶስ።</persName></q>
+</item>
+
+<item xml:id="a3">
+    <locus target="#1r"/>
+    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of <placeName ref="INS0101MadhaneAlam">Madḫāne ʿAlam</placeName>
+    </desc>
+    <q xml:lang="gez">ስንክሳር፡ ዘቅ' ፡ መ' ፡ ዓለም።</q>
```

Comment:
It is not possible to know from the catalogue, but ቅ' ፡ መ' ፡ is Wright's abbreviation. It's always spelled out in the manuscripts, so it can be safely spelled out here as well. In this manuscript:
![image](https://user-images.githubusercontent.com/19650382/100428915-e201e480-3094-11eb-9685-09dcb5e22717.png)